CodeSniffer/Standards/Generic/Sniffs/PHP/SyntaxSniff.php

```diff
@@ -74,11 +74,11 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
         }
 
         $fileName = $phpcsFile->getFilename();
-        $cmd      = $this->_phpPath." -l \"$fileName\" 2>&1";
+        $cmd      = $this->_phpPath." -l -d error_prepend_string='' \"$fileName\" 2>&1";
```

Comment:
The added `m` in the regex already makes it less error prone as it will match the `^` in the regex for start of line instead of start of string, so it will match as soon as it finds a line which contains a (parse) error message ;-)

That's what I meant by making _"double sure"_ :heart_eyes: